documentation/source/building.rst

```diff
@@ -273,3 +273,19 @@ Additional Environment Variables
     Path to the directory containing the cocotb Makefiles and simulator libraries in the subdirectories
     :file:`lib`, :file:`include`, and :file:`makefiles`.
     You don't normally need to modify this.
+
+.. envvar:: GPI_EXTRA
+
+    A comma-separated list of extra libraries that are dynamically loaded at runtime.
+    A function from each of these libraries will be called prior to elaboration, allowing these libraries to register
```

Comment:
```suggestion
    A function from each of these libraries will be called as an entry point prior to elaboration, allowing these libraries to register
```
Should probably introduce the concept of entry point before giving the examples.